media/js/firefox/whatsnew/whatsnew-70.js

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var client = Mozilla.Client;
+
+    function handleOpenProtectionReport(e) {
+        e.preventDefault();
+        Mozilla.UITour.showProtectionReport();
+
+        window.dataLayer.push({
+            'event': 'in-page-interaction',
+            'eAction': 'link click',
+            'eLabel': 'See what Firefox has blocked for you'
+        });
+    }
+
+    function handleOpenLockwise(e) {
+        e.preventDefault();
+        Mozilla.UITour.showHighlight('logins');
+
+        window.dataLayer.push({
+            'event': 'in-page-interaction',
+            'eAction': 'link click',
+            'eLabel': 'Update your Firefox browser'
+        });
+    }
+
+    if (client.isFirefoxDesktop) {
+        if (client._getFirefoxMajorVersion() >= 70) {
+            // show "See what Firefox has blocked for you" links.
+            document.querySelector('main').classList.add('state-firefox-desktop-70');
+
+            // Intercept link clicks to open about:protections page using UITour.
+            Mozilla.UITour.ping(function() {
+                var protectionReportLinks = document.querySelectorAll('.js-open-about-protections');
+                var lockwiseLinks = document.querySelectorAll('.js-open-lockwise');
+
+                // For UI Tour CTAs, we will treat these more as an action rather than a page navigation.
+                for (var i = 0; i < protectionReportLinks.length; i++) {
+                    protectionReportLinks[i].setAttribute('role', 'button');
```

Comment:
I flip-flopped on this multiple times when working on the privacy/products page, but in the end I decided to omit `role="button"` here. 

I think I agree with your original thought that this does feel more like a destination, rather than an action. It just happens to be an in-product page, instead of one on the www.

One thing I did do to help solidify this was to add `protectionReportLinks[i].href = 'about:protections';`, which overwrites the existing sumo links, and gives better context to where the link now points to. Worth adding here instead?

Reply:
Sounds good. I think I will keep the current logic for the lockwise links because that does feel like more of an action vs. a destination.